src/parsers/pm_io/psse.jl

```diff
@@ -899,84 +1727,124 @@ function _psse2pm_dcline!(pm_data::Dict, pti_data::Dict, import_all::Bool)
             if import_all
                 _import_remaining_keys!(sub_data, dcline)
             end
-
+            branch_isolated_bus_modifications!(pm_data, sub_data)
             push!(pm_data["dcline"], sub_data)
         end
     end
 
     if haskey(pti_data, "VOLTAGE SOURCE CONVERTER")
-        @info(
-            "VSC-HVDC lines are supported via a dc line model approximated by two generators and an associated loss."
-        )
         for dcline in pti_data["VOLTAGE SOURCE CONVERTER"]
             # Converter buses : is the distinction between ac and dc side meaningful?
-            dcside, acside = dcline["CONVERTER BUSES"]
+            from_bus, to_bus = dcline["CONVERTER BUSES"]
 
             # PowerWorld conversion from PTI to matpower seems to create two
             # artificial generators from a VSC, but it is not clear to me how
             # the value of "pg" is determined and adds shunt to the DC-side bus.
             sub_data = Dict{String, Any}()
+            sub_data["name"] = strip(dcline["NAME"], ['"', '\''])
 
             # VSC intended to be one or bi-directional?
-            sub_data["f_bus"] = pop!(dcside, "IBUS")
-            sub_data["t_bus"] = pop!(acside, "IBUS")
+            sub_data["f_bus"] = from_bus["IBUS"]
+            sub_data["t_bus"] = to_bus["IBUS"]
+            if pm_data["has_isolated_type_buses"]
+                push!(pm_data["connected_buses"], sub_data["f_bus"])
+                push!(pm_data["connected_buses"], sub_data["t_bus"])
+            end
             sub_data["br_status"] =
-                if pop!(dcline, "MDC") == 0 ||
-                   pop!(dcside, "TYPE") == 0 ||
-                   pop!(acside, "TYPE") == 0
+                if dcline["MDC"] == 0 ||
+                   from_bus["TYPE"] == 0 ||
+                   to_bus["TYPE"] == 0
                     0
                 else
                     1
                 end
+            sub_data["available"] = sub_data["br_status"] == 0 ? false : true
```

Comment:
see above, the thing before the `?` is already Boolean so you can just work with that